proto/buf/alpha/audit/v1alpha1/envelope.proto

```diff
@@ -0,0 +1,561 @@
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_RECOMMENDED_REPOSITORIES = 63;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_RECOMMENDED_TEMPLATES = 64;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_SET_RECOMMENDED_REPOSITORIES = 65;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_SET_RECOMMENDED_TEMPLATES = 66;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_REFERENCE_BY_NAME = 67;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_CREATE_REPOSITORY_BRANCH = 68;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_REPOSITORY_BRANCHES = 69;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_REPOSITORY_COMMITS_BY_BRANCH = 70;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_REPOSITORY_COMMITS_BY_REFERENCE = 71;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_REPOSITORY_COMMIT_BY_REFERENCE = 72;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_REPOSITORY_COMMIT_BY_SEQUENCE_ID = 73;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_CREATE_REPOSITORY_TAG = 74;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_REPOSITORY_TAGS = 75;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_REPOSITORY = 76;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_REPOSITORY_BY_FULL_NAME = 77;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_REPOSITORIES = 78;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_USER_REPOSITORIES = 79;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_REPOSITORIES_USER_CAN_ACCESS = 80;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_ORGANIZATION_REPOSITORIES = 81;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_CREATE_REPOSITORY_BY_FULL_NAME = 82;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_DELETE_REPOSITORY = 83;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_DELETE_REPOSITORY_BY_FULL_NAME = 84;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_DEPRECATE_REPOSITORY_BY_NAME = 85;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_UNDEPRECATE_REPOSITORY_BY_NAME = 86;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_REPOSITORIES_BY_FULL_NAME = 87;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_MODULE_PINS = 88;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_LOCAL_MODULE_PINS = 89;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_SEARCH = 90;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_CREATE_TOKEN = 91;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_TOKEN = 92;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_DELETE_TOKEN = 93;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_TOKENS = 94;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_CREATE_USER = 95;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_USER = 96;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_GET_USER_BY_USERNAME = 97;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_USERS = 98;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_LIST_ORGANIZATION_USERS = 99;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_DELETE_USER = 100;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_DEACTIVATE_USER = 101;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_UPDATE_USER_SERVER_ROLE = 102;
+  ACTION_BUF_ALPHA_REGISTRY_V1_ALPHA1_COUNT_USERS = 103;
+  ACTION_BUF_ALPHA_REGISTRYINTERNAL_V1_ALPHA1_CREATE_PLUGIN_VERSION = 104;
+  ACTION_BUF_ALPHA_REGISTRYINTERNAL_V1_ALPHA1_DELETE_PLUGIN_VERSION = 105;
+  ACTION_BUF_ALPHA_REGISTRYINTERNAL_V1_ALPHA1_AUTHENTICATE_TOKEN = 106;
```

Comment:
It's worth noting that this includes the internal API names as well, but obviously no way to access them.